spec/controllers/reservations_controller_spec.rb

```diff
@@ -6,28 +6,27 @@
   render_views
 
   before(:all) do
-    @app_config = FactoryGirl.create(:app_config)
-
     @user = FactoryGirl.create(:user)
     @banned = FactoryGirl.create(:banned)
     @checkout_person = FactoryGirl.create(:checkout_person)
     @admin = FactoryGirl.create(:admin)
   end
 
   after(:all) do
-    User.delete_all
-    AppConfig.delete_all
-    Reservation.delete_all
-    Category.delete_all
-    EquipmentModel.delete_all
```

Comment:
This might not even be necessary, it was something I suspected early on but I'm not sure if it ended up actually causing any errors. In any case, it's safer.

I think I'm just going to get rid of this block and have the database cleaner run for every `after(:all)` (see [here](http://renderedtext.com/blog/2012/10/10/cleaning-up-after-before-all-blocks/)). Because we persist so much data in our tests, I think that it's easier to have a more complicated cleaning setup for now and keep some of the `before(:all)` blocks for efficiency until #1412